src/zadaci.js

```diff
@@ -0,0 +1,54 @@
+// Funkcija za kreiranje tabele pomocu innerhtml-a
+
+function kreiranjeTabele1(brojRedova, brojKolona){
+  let html = "<table>";
+
+  for (let i=0; i<brojRedova; i++){
+    html += "<tr>";
+    for (let j=0; j<brojKolona; j++){
+      html += "<td>" + Math.ceil(Math.random()*20) + "</td>";
+    }
+    html += "</tr>";
+  }
+
+  html += "</table>";
+
+  document.getElementById("id_tabela1").innerHTML = html;
+}
+
+
+
+// Funkcija za Kreiranje tabele pomocu AppendChild
+
+function kreiranjeTabele2(brojRedova, brojKolona){
+  let newTR;
+  let newTD;
+  let tabela = document.createElement("table");
+
+  for (let i=0; i<brojKolona; i++){
+    newTD = document.createElement("td");
+
+    for (let j=0; j<brojRedova; j++){
+      newTR = document.createElement("tr");
```

Comment:
Ako se promenljive koristi samo u ovom bloku onda bi trebalo da stoji `let newTr =` a ono gore na početku let izbriši.
Uz to da si uradio `npm i` na početku ovde bi ti linter rekao da promeniš u const.